ee/packages/omnichannel-services/src/OmnichannelTranscript.ts

```diff
@@ -323,45 +328,58 @@ export class OmnichannelTranscript extends ServiceClass implements IOmnichannelT
 				translations,
 			};
 
-			await this.doRender({ template, data, details });
+			await this.doRender({ data, details });
 		} catch (error) {
 			await this.pdfFailed({ details, e: error as Error });
 		} finally {
 			this.currentJobNumber--;
 		}
 	}
 
-	async doRender({ template, data, details }: { template: Templates; data: WorkerData; details: WorkDetailsWithSource }): Promise<void> {
+	async promisifyStream(stream: NodeJS.ReadableStream, buffer: Uint8Array[]) {
```

Comment:
maybe you can use this one instead? https://github.com/RocketChat/Rocket.Chat/blob/eb40ccef9f9672e0ade130d429040efa2a0c20c4/apps/meteor/app/file-upload/server/lib/streamToBuffer.ts#L3

maybe move it to a lib? wdyt?

Reply:
All right, moved the function to `tools` package, but won't refactor the other places on this PR 👀 

but now we have a central func for doing so.